sys-devel/distcc/distcc-3.4-r7.ebuild

```diff
@@ -0,0 +1,212 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..13} )
+
+inherit autotools flag-o-matic prefix python-single-r1 systemd xdg
```

Comment:
Adding "xdg" here and later in the pkg_-functions should probably be its own patch. It was a drive-by find and is valid on its own.

Would mean several revbumps but i think i will do that.